packages/shared/src/hooks/analytics/useAnalyticsContextData.ts

```diff
@@ -14,6 +14,8 @@ const generateEventId = (now = new Date()): string => {
   return `${timePart}${randomStr}`;
 };
 
+export type PostOrigin = 'article page' | 'article modal';
```

Comment:
I think it's a duplicate

Reply:
I've kept this one and removed the other one now. Thank you for the notice!